examples/echo-compat/handlers.go

```diff
@@ -36,12 +36,6 @@ func fuegoControllerPost(c fuego.ContextWithBody[HelloRequest]) (*HelloResponse,
 	}, nil
 }
 
-func serveOpenApiJSONDescription(s *fuego.OpenAPI) echo.HandlerFunc {
-	return func(ctx echo.Context) error {
-		return ctx.JSON(http.StatusOK, s.Description())
-	}
-}
```

Comment:
Wasn't it detected by unused linter?

Reply:
no actually. I only noticed from my IDEs linter